src/p5.dance.js

```diff
@@ -20,37 +20,6 @@ const ANIMATIONS = {};
 const FRAMES = 24;
 const METADATA = {};
 
-const songs = {
-  macklemore: {
-    url: 'https://curriculum.code.org/media/uploads/chu.mp3',
-    bpm: 146,
-    delay: 0.2, // Seconds to delay before calculating measures
-    verse: [26.5, 118.56], // Array of timestamps in seconds where verses occur
-    chorus: [92.25, 158] // Array of timestamps in seconds where choruses occur
-  },
-  macklemore90: {
-    url: 'https://curriculum.code.org/media/uploads/hold.mp3',
-    bpm: 146,
-    delay: 0.0, // Seconds to delay before calculating measures
-    verse: [0, 26.3], // Array of timestamps in seconds where verses occur
-    chorus: [65.75] // Array of timestamps in seconds where choruses occur
-  },
-  hammer: {
-    url: 'https://curriculum.code.org/media/uploads/touch.mp3',
-    bpm: 133,
-    delay: 2.32, // Seconds to delay before calculating measures
-    verse: [1.5, 15.2], // Array of timestamps in seconds where verses occur
-    chorus: [5.5, 22.1] // Array of timestamps in seconds where choruses occur
```

Comment:
Should the "verse" and "chorus" metadata be added to https://levelbuilder-studio.code.org/api/v1/sound-library/hoc_song_meta/hammer.json?

Reply:
Good catch. I know JoshS is going to be working on finding these timestamps, so we'll probably have to find a better way to get it in metadata, but in the meantime, I'll just copy them over.

Reply:
Updated the current S3 files with these values.